src/app/components/Ionic/IonicProvider.tsx

```diff
@@ -0,0 +1,38 @@
+import { createContext, FC, PropsWithChildren, useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { Capacitor } from '@capacitor/core'
+import { App } from '@capacitor/app'
+
+const IonicContext = createContext<undefined>(undefined)
```

Comment:
Why are we using React context for attaching listeners? it's not going to be used with always being undef

Reply:
I figured there will be some Ionic state specific variables coming in the future, and we will need Context anyway. Like device theme probably - to minimize entanglement with web and extension platform. I can convert it to component, but I think we will run into issues sooner than later.